ajax/libs/froala-design-blocks/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "froala-design-blocks",
+  "description": "An amazing collection of over 170 ready to use responsive design blocks based on Bootstrap.",
+  "homepage": "https://www.froala.com/design-blocks",
+  "keywords": [
+    "froala",
+    "froala design blocks",
+    "html design",
+    "bootstrap",
+    "material design",
+    "html templates",
+    "website design",
+    "html5",
+    "trending design"
+  ],
+  "license": "https://github.com/froala/design-blocks/blob/master/LICENSE",
```

Comment:
@extend1994 @sufuf3 I forget how we dill with custom license. Do we use a object for it? Thanks.

Reply:
Looking forward to updates so that I can make the necessary changes. Thanks in advance.

Reply:
@maruilian11 @extend1994 @sufuf3 any suggestions on this? Thanks.

Reply:
@PeterDaveHello According to [npm package.json doc](https://docs.npmjs.com/files/package.json), using objects for licenses has been deprecated. Now the way for a license that hasn't been assigned an SPDX identifier is:
``` 
"license" : "SEE LICENSE IN <filename>"
```
But that means we may need to add custom license file to cdnjs, too.
Can you give me some advice? Thanks.

Reply:
Just use:
```js
"license": {
  "type": "Custom license",
  "url": "https://github.com/froala/design-blocks/blob/master/LICENSE"
}
```

we are not npmjs ...

Reply:
@stefanneculai so is it kind of open source license?

Reply:
That is correct. I'll update the license shortly.

Reply:
Thanks!